src/components/grader/DownloadResult.js

```diff
@@ -1,27 +1,26 @@
 import React from "react";
 import fileDownload from "js-file-download";
-//import apiClient from "../../api-client";
 import { Button, Header } from "semantic-ui-react";
 import GradingService from '../../services/GradingService'
 
 function DownloadResult(props) {
-  function downloadFile() {
-    GradingService
-      .downloadResults(props.course, props.lab).then(res => {
-        fileDownload(res, "results.zip");
-      });
-  }
+    function downloadFile() {
+        GradingService
+            .downloadResults(props.course, props.lab).then(res => {
+                fileDownload(res, "results" + new Date().valueOf() + ".zip");
```

Comment:
this name should have an `_` separator (between results and the timestamp). but that's not a major thing